utils/src/test/java/com/cloud/utils/SwiftUtilTest.java

```diff
@@ -90,4 +96,134 @@ public void testGetContainerName(){
 
         assertEquals(expected, output);
     }
+
+    @Test
```

Comment:
Unit Tests! That is awesome!!
Thank you very much for these. We need to create a culture of writing them.

Reply:
I totally agree, but all the credit for this PS should go to @pdube. I only revived and rebased his PR.